### DISCUSSION TITLE
Bug report of github itself?

### DISCUSSION THREAD

**Discussion by piradata** [community/community#54348] [Other Feature Feedback, Questions, & Ideas]
Hello, do you guys know where is the correct place to open a bug report/issue about github itselt know that the [old repo](https://github.com/isaacs/github) is dead?

Tried to open one [here](https://github.com/orgs/community/discussions/53086) but it seems to be the wrong place/category...

**Comment by piradata** [comment]
nevermind, found it :) https://github.com/orgs/community/discussions/new?category=issues-and-projects

**Comment by tuves** [comment]
Hi @piradata,

Please post the bug report in the product category that it belongs with, with the `bug report` label. [Issues and Projects](https://github.com/orgs/community/discussions/new?category=issues-and-projects) is for [GitHub Issues and Projects](https://github.com/features/issues) product-related posts.

**Reply by piradata** [reply]
So, only things related to project planning go there?

Also, how do i add a label on [here](https://github.com/orgs/community/discussions/53086)? tried commenting a `\label bug report` but didn't worked..
